pkg/apis/discovery/types.go

```diff
@@ -34,8 +34,8 @@ type EndpointSlice struct {
 	// addressType specifies the type of address carried by this EndpointSlice.
 	// All addresses in this slice must be the same type. The following address
 	// types are currently supported:
```

Comment:
Add a comment someehere that this field is immutable after creation

Reply:
Done, thanks!